marlowe-playground-client/src/MainFrame/View.purs

```diff
@@ -187,6 +189,17 @@ menuBar state =
     else
       menuButton (OpenModal GithubLogin) name
 
+  -- Even if we end up writting more by selecting the cases in which the buttons should
+  -- appear, I prefer this to selecting which views we shouldn't show the buttons. The
+  -- reason is that if we add a new view, we don't need to adjust this.
+  -- TODO: Check if we should show the buttons for WalletEmulator and ActusBlocklyEditor
```

Comment:
Good point, it seems the ACTUS Labs currently only saves Haskell, Javascript, and Marlowe. So better not to show it for now.
I cannot even find a way to access the Wallet, so it probably shouldn't have them either.

Reply:
you can access wallets with the URL `/#/wallets`

Reply:
but I agree, shouldn't be visible on those

Reply:
how about ActusBlocklyEditor?

Reply:
@shmish111  while doing the home page redesign, I found that on line 121-138 there is a `otherActions` helper that also appends items to the topBar depending on the view. Do you think it makes sense in a following PR to move that inside `menuBar`?